lib/utils.ts

```diff
@@ -215,9 +216,7 @@ const windowOpen = (
           onClose(shareDialog);
         }
       } catch (e) {
-        /* eslint-disable no-console */
-        console.error(e);
-        /* eslint-enable no-console */
+        logger.error('Window open', e);
```

Comment:
Nit pick but maybe match function names and casing so it's easy to search and find in code base

So `windowOpen:` instead of `Window open`